shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -1288,31 +1288,39 @@ fn test_batch_statements(connection: &Session) {
     {
         let mut batch = Batch::new(BatchType::LOGGED);
         for i in 0..2 {
-            let statement = format!("INSERT INTO batch_keyspace.batch_table (id, lastname, firstname) VALUES ({}, 'text', 'text')", i);
+            let statement = format!("INSERT INTO batch_keyspace.batch_table (id, lastname, firstname) VALUES ({}, 'text1', 'text2')", i);
             batch.add_statement(&stmt!(statement.as_str())).unwrap();
         }
         connection.execute_batch(&batch).wait().unwrap();
         assert_query_result(
             connection,
-            "SELECT lastname FROM batch_keyspace.batch_table WHERE id = 1;",
-            &[&[ResultValue::Varchar("text".into())]],
+            "SELECT id, firstname, lastname FROM batch_keyspace.batch_table WHERE id = 1;",
+            &[&[
+                ResultValue::Int(1),
```

Comment:
And every row?

Reply:
still need this in case you missed it.

Reply:
Have I fixed everything you meant now?

Reply:
I'm thinking to remove the WHERE and add an ORDER BY and then assert we get back both rows.

Reply:
We don't need to ORDER BY, the assert methods we have sort it for us.

Reply:
oh excellent, lets remove the WHERE then.

The reason being: we arent trying to test select query handling, we are trying to test the batch successfully ran ALL inserts.